### PR TITLE
Three way frequencies

### PR DESCRIPTION
**Pull request by AlexSananka**
@africanmathsinitiative/developers  This is in progress

### CODE REVIEW

**Comment by AlexSananka** [comment]
@dannyparsons you can merge in this for now

File: instat/dlgThreeVariableFrequencies.vb [hunk 1]
```diff
@@ -1,3 +1,260 @@
...
+        ucrPnlFrequencyDisplay.AddRadioButton(rdoGraph)
+
+        ucrChkWeights.Enabled = False
+        cmdOptions.Enabled = False
+
+        ucrPnlFrequencyDisplay.AddFunctionNamesCondition(rdoTable, "sjtab")
+        ucrPnlFrequencyDisplay.AddFunctionNamesCondition(rdoGraph, "sjplot")
+
+        ucrPnlFreqType.AddParameterValuesCondition(rdoCount, "fun", "grpfrq")
+        ucrPnlFreqType.AddParameterValuesCondition(rdoCount, "margin", False)
```
Comment (dannyparsons) [inline instat/dlgThreeVariableFrequencies.vb]: This doesn't quite look right. What is the condition you wanted here?

Comment (AlexSananka) [inline instat/dlgThreeVariableFrequencies.vb]: Yes sure you're right,I should have edited this.At first what I wanted to achieve here is like telling the code that margin is not associated with count,But I think it is already by default.I think we're doing this  when we want extra bars for either column,rows and cells.
I will delete this.

Comment (dannyparsons) [inline instat/dlgThreeVariableFrequencies.vb]: I think what you want is a parameter present condition. This is when the condition is about whether the parameter is in the code and not about the value. This is what @stevenndungu did for two variable dialog.

**Comment by AlexSananka** [comment]
@dannyparsons This is ready

**Pull request merged**